packages/@okta/vuepress-site/docs/guides/social-login/main/paypal/create-app.md

```diff
@@ -1 +1 @@
-[Create an app](https://developer.paypal.com/docs/api-basics/manage-apps/#create-or-edit-sandbox-and-live-apps) and [configure it](https://developer.paypal.com/docs/log-in-with-paypal/integrate/#enable-log-in-with-paypal) at <StackSelector snippet="idp" noSelector inline />.
\ No newline at end of file
+[Create an app](https://developer.paypal.com/docs/api-basics/manage-apps/#create-or-edit-sandbox-and-live-apps) and [configure it](https://developer.paypal.com/docs/log-in-with-paypal/integrate/#enable-log-in-with-paypal) at <StackSnippet snippet="idp" inline />.
```

Comment:
Can we remove create-app.md and put the specific Paypal info into apptype.md and follow the other IdP section with: 
`Create and register <StackSnippet snippet="apptype" inline /> at <StackSnippet snippet="idp" inline />.`

Reply:
This will be addressed in another PR for OKTA-459709.